docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+services:
+  db:
+    build: ./simulation_generator/database/
```

Comment:
This path should be `./database` and other subsequent paths for database should be changed